text/0061-error-format-and-definitions.md

```diff
@@ -122,6 +122,80 @@ HTTP Code: `400 Bad Request`
 
 ---
 
+## api_key_already_exists
+
+`Synchronous`
+
+### Context
+
+This error happens when a user tries to create an API Key that already exists for the given `uid`.
+
+### Error Definition
+
+HTTP Code: `409 Conflict`
+
+```json
+{
+    "message": "`uid` field value `:value` already exists for an API key.",
```

Comment:
related to https://github.com/meilisearch/meilisearch/pull/2438#discussion_r885547329

```suggestion
    "message": "`uid` field value `:value` is already an existing API key.",
```